bench/wtperf/wtperf.c

```diff
@@ -644,8 +656,11 @@ worker(void *arg)
 			trk = &thread->read;
 			/* FALLTHROUGH */
 		case WORKER_UPDATE:
+		case WORKER_MODIFY:
 			if (*op == WORKER_UPDATE)
 				trk = &thread->update;
+			if (*op == WORKER_MODIFY)
+				trk = &thread->modify;
```

Comment:
Alphabetize the case to sort above `WORKER_UPDATE` and alphabetize the conditional in the same way.

Reply:
corrected.